app/main.py

```
@@ -1,7 +1,12 @@
 from fastapi import FastAPI
+from app.api import vector, recommend
+from app.database.connection import Base, engine
```

Ruff (0.12.2)

`app.database.connection.Base` imported but unused

Remove unused import

(F401)